x-pack/metricbeat/module/mssql/fields.go

```diff
@@ -16,7 +16,8 @@ func init() {
 	}
 }
 
-// AssetMssql returns asset data
+// AssetMssql returns asset data.
+// This is the base64 encoded gzipped contents of /Users/ruflin/Dev/gopath/src/github.com/elastic/beats/x-pack/metricbeat/module/mssql.
```

Comment:
Can you modify something so that this isn't any absolute path, but instead is relative to the beat's directory like `./module/mssql/fields.go`. I didn't test to see if this would work, but try changing this line to not construct an abs path.

https://github.com/elastic/beats/blob/e92c6b8f2c6d954d64c89e0c97dbbe7b87b19644/dev-tools/mage/fields.go#L104

Reply:
Seems to do the trick but need to do some more testing.